docs/scenarios_authoring/HandlingSchemas.md

```diff
@@ -0,0 +1,113 @@
+---
+sidebar_position: `11
+---`
+
+# Handling Schemas
+
+## Overview
+
+Nussknacker supports two types of schema:
+* [JSON Schema](https://json-schema.org/) in version: `Draft 7`
+* [Avro Schema](https://avro.apache.org/) in version: `1.11.0`
+
+Avro Schema nad JSON Schema are available on [Streaming Source/Sik](/docs/scenarios_authoring/DataSourcesAndSinks.md) 
+in contrast to [RequestResponse Source/Sink](/docs/scenarios_authoring/RRDataSourcesAndSinks.md) where available is just only one - JSON Schema.
+
+Nussknacker converts `source schema` to own static `Typing` information to create hinting on the FE Designer,
+whereas `sink schema` is used to validation output data.
+
+## Source conversion
+Conversion at srouce to specific type means that behind the scene Nussknacker converts `primitive type` to `logical type`,
```

Comment:
from source?

Reply:
Conversion is done in some place, so I think there should be `at` or `on`?